fontc/src/lib.rs

```diff
@@ -859,16 +816,12 @@ mod tests {
         source: &str,
         adjust_args: impl Fn(&mut Args),
     ) -> TestCompile {
-        let temp_dir = tempdir().unwrap();
-        let build_dir = temp_dir.path();
-        let mut args = Args::for_test(build_dir, source);
-        adjust_args(&mut args);
-        let result = compile(args);
+        let result = TestCompile::compile(source, |mut args| {
+            adjust_args(&mut args);
+            args
+        });
```

Comment:
if this fn took the same closure type as the one on `TestCompile::compile` (this one mutates args, the other one takes ownership and returns them) then you could just pass the closure straight through here, like, `TestCompile::compile(source, adjust_args);` which is a bit cleaner.

Reply:
I don't know why I made it different! Good catch.